test/Sentry.Tests/SentryOptionsTests.verify.cs

```diff
@@ -18,9 +18,12 @@ public Task Integrations_default_ones_are_properly_registered()
         };
         Hub _ = new(options, Substitute.For<ISentryClient>());
 
-        return Verify(logger.Entries)
-            .UniqueForOSPlatform()
+        var settingsTask = Verify(logger.Entries)
+            .UniqueForTargetFrameworkAndVersion()
             .UniqueForRuntime()
             .AutoVerify(includeBuildServer: false);
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            settingsTask = settingsTask.UniqueForOSPlatform();
```

Comment:
We it's only unique in that instance. If it was ever unique on any of the other platforms, our verify tests would start failing on those platforms. Until then though, they all drop back to the "default" verify received file. We could make it unique for all OS Platforms but it would just results in a bunch of extra/redundant verify files (that were all the same).